contracts/test/SafeSecretHarbour.spec.ts

```diff
@@ -22,20 +28,186 @@ describe("SafeInternationalHarbour", () => {
 
 		const { decryptionKey, encryptionKey } = await randomX25519KeyPair();
 
-		return { deployer, signer, alice, harbour, chainId, safe, decryptionKey, encryptionKey };
+		return { deployer, signer, notary, alice, harbour, chainId, safe, decryptionKey, encryptionKey };
 	}
 
+	it("should act as an encrypted transaction queue", async () => {
```

Comment:
@rmeissner and @remedcu - I added a comprehensive E2E like demonstration of how I envision secret harbour to be used by clients.